Tx.ts

```diff
@@ -0,0 +1,274 @@
+/**
```

Comment:
Er what is this file doing here? This file corresponds to the `V1/Tx.hs` Plutus Ledger API module which already exists in `src/Lib/V1/Tx.ts` https://github.com/mlabs-haskell/plutus-ledger-api-typescript/blob/main/src/Lib/V1/Tx.ts.

Perhaps this got duplicated somehow? I'd guess this probably isn't necessary.

Reply:
oops, I just recently transitioned to a GUI IDE, and dragging files randomly :) maybe sticking to vim is not a bad idea